common/app/common/pages/RatingData.java

```diff
@@ -0,0 +1,24 @@
+package common.pages;
+
+import common.cms.CmsPage;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+public class RatingData {
+    private final CmsPage cms;
+
+    public RatingData(final CmsPage cms) {
+        this.cms = cms;
+    }
+
+    public List<SelectableData> getRating() {
+        return asList(
+                new SelectableData("5 Stars", "5", cms.getOrEmpty("ratingFiveStarText"), "", false),
```

Comment:
@trangng I have actually a doubt here. In the final shape of Sunrise, what kind of messages should go to CMS and what kind should not be modifiable by the merchant? Last time I asked it was generally "everything", but probably there will be things we don't want to. We need to know exactly how to draw the line.